roles/foreman_provisioning/tasks/configure_centos_7.yml

```diff
@@ -1,21 +1,21 @@
 - name: 'set CentOS medium name'
   set_fact:
-    centos_medium_name: "{{ 'CentOS 7 mirror' if foreman_provisioning_foreman_version == 'nightly' or (foreman_provisioning_foreman_version is version_compare('2.1', '>=')) else 'CentOS mirror' }}"
+    centos_medium_name: "{{ 'CentOS 9 mirror' if foreman_provisioning_foreman_version == 'nightly' or (foreman_provisioning_foreman_version is version_compare('2.1', '>=')) else 'CentOS mirror' }}"
```

Comment:
The correct name is `CentOS Stream 9 mirror`:
https://github.com/theforeman/foreman/blob/e78bd159584f2f8d09b3d57ab94615a2ae26ecb7/db/seeds.d/100-installation_media.rb#L18-L22